packages/plugin-version/sources/commands/version/check.tsx

```diff
@@ -91,9 +91,9 @@ export default class VersionApplyCommand extends Command<CommandContext> {
         <Box>
           {strategies.map(strategy => {
             if (strategy === decision) {
-              return <Box key={strategy} paddingLeft={2}><Color green>◼</Color> {strategy}</Box>;
+              return <Box key={strategy} paddingLeft={2}><Color green>◉</Color> {strategy}</Box>;
             } else {
-              return <Box key={strategy} paddingLeft={2}><Color yellow>◻</Color> {strategy}</Box>;
```

Comment:
I'm fine with this change, but I'm a bit confused because I see them as unicode characters, not emojis 🤔

Reply:
Try copying the char into google: https://www.google.com/search?q=%E2%97%BC

Reply:
Its this emoji: https://emojipedia.org/black-medium-square/

Reply:
Here is somebody having a problem with it: https://discordapp.com/channels/226791405589233664/654372321225605128/709074102186672282